kv/test_raftstore/scheduler.go

```diff
@@ -395,11 +395,12 @@ func (m *MockSchedulerClient) tryFinished(op *Operator, region *metapb.Region, l
 			for _, p := range region.GetPeers() {
 				if add.peer.GetId() == p.GetId() {
 					add.pending = true
-				} else {
-					// TinyKV rejects AddNode.
+					op.Data = add
```

Comment:
nice catch. how about making `op.Data` as a pointer

Reply:
sure,  I change addPeer op data to pointer